src/shogun/features/RandomKitchenSinksDotFeatures.cpp

```diff
@@ -37,14 +38,11 @@ SGMatrix<float64_t> RandomKitchenSinksDotFeatures::generate_random_coefficients(
 {
 	SGVector<float64_t> vec = generate_random_parameter_vector();
```

Comment:
actually no, you are generating the vec once and then throw it away without looking at it

Reply:
I was gonna change it anyway 😅. I made a review on it remember?

Reply:
So nothing besides this, then

Reply:
vec.vlen has num_feature_dim_space + 1  i.e. num_samples  + 1 . So should I hardcode that? in 
```
random_params(num_samples + 1 , num_samples)
```

Reply:
what I mean is that you create the vector and the you overwrite it in the first iteration of the loop